lib-ce/emqx_modules/src/emqx_mod_delayed.erl

```diff
@@ -123,6 +128,12 @@ on_message_publish(Msg = #message{
 on_message_publish(Msg) ->
     {ok, Msg}.
 
+check_acl(ClientInfo, PubSub, <<"$delayed/", Topic/binary>>, AclResult) ->
+    [_Delay, Topic1] = binary:split(Topic, <<"/">>),
```

Comment:
Agree, refusing the malformed topic is a good option

Reply:
I thought about it and it doesn't seem necessary, because messages which topic starting with `$delayed` will be validated by emqx_mod_delayed:on_message_publish